text/0030-protocol-messages.md

```diff
@@ -0,0 +1,242 @@
++ Feature name: `protocol-messages`
++ Start date: 2020-04-27
++ RFC PR: [iotaledger/bee-rfcs#30](https://github.com/iotaledger/bee-rfcs/pull/30)
++ Bee issue: [iotaledger/bee#70](https://github.com/iotaledger/bee/issues/70)
+
+# Summary
+
+This RFC introduces the IOTA protocol messages that were initially added in
+[IRI#1393](https://github.com/iotaledger/iri/pull/1393).
+
+# Motivation
+
+To be able to take part in the IOTA networks, Bee nodes need to implement the exact same protocol presented in this RFC
+and currently being used by [IRI](https://github.com/iotaledger/iri) nodes and
+[HORNET](https://github.com/gohornet/hornet) nodes. However, it does not necessarily mean implementing the same versions
+of the protocol and a design decision - later explained - concludes that Bee nodes and IRI nodes will not be able to
+communicate with each other.
```

Comment:
This sentense is very confusing, It's a double negation. Not sure if on purpose.

Reply:
Where is the double negation ?

Reply:
oh, I see. But in that case, there should be a comma after `of the protocol`. 

I suggest making it 2 sentences:
However, it does not necessarily mean implementing the same version of the protocol.
A design decision, further detailed down below, concludes, that Bee nodes and IRI nodes will not be able to communicate with each other.

Reply:
Alright, will do 👍